applications/StructuralMechanicsApplication/custom_elements/shell_thin_element_3D3N.hpp

```diff
@@ -179,6 +181,12 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ShellThinElement3D3N
     {
     }
 
+    // by M.Fusseder

```

Comment:
This does not follow Doxygen notation

Reply:
take the name out. (Add yourself to the list of authors if the change is significative)